### PR TITLE
[#44] Add license file (main)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2023, The University of North Carolina at Chapel Hill
+All rights reserved.
```
Comment (alanking) [inline LICENSE:4]: Should this be 2024? Noticed the audit one says 2016...

Comment (korydraughn) [reply LICENSE:4]: A quick google search states it's supposed to match the year the work was first published.

Will update this and all other PRs.

Comment (trel) [reply LICENSE:4]: first commit was 2019

Comment (korydraughn) [reply LICENSE:4]: done

**Review by alanking — APPROVED** [review]

**Comment by korydraughn** [comment]
#'d

**Pull request merged**